src/bootstrap/src/bin/rustdoc.rs

```diff
@@ -59,8 +59,6 @@ fn main() {
     if stage == "0" {
         cmd.arg("--cfg=bootstrap");
     }
-    cmd.arg("-Zunstable-options");
-    cmd.arg("--check-cfg=cfg(bootstrap)");
```

Comment:
Why was the `--check-cfg` flag removed?

Reply:
Duplicated as well.

Reply:
Verified with `--dump-bootstrap-shims`. Thanks!